snappy/poedit-git/snap/snapcraft.yaml

```diff
@@ -0,0 +1,176 @@
+      #- libboost-dev
+      #- libboost-iostreams-dev
+      #- libboost-regex-dev
+      #- libboost-system-dev
+      #- libboost-thread-dev
+      - libcld2-dev
+      - libcpprest-dev
+      - libdb++-dev
+      - libgtk2.0-dev
+      - libgtkspell-dev
+      - libicu-dev
+      - liblucene++-dev
+      - libsecret-1-dev
+      # wxWidgets library provided by Ubuntu 16.04 is too old
+      #- libwxgtk3.0-dev
+      #- nlohmann-json-dev
+      - xsltproc
+      - xmlto
+      - zip
+    override-build: |
+      ./bootstrap
+      snapcraftctl build
+    stage-packages:
+      - gettext
+    filesets:
+      executables:
+        - usr/local/bin/*
+      freedesktop-desktop-entries:
+        - usr/local/share/applications/*
+      icons:
+        - usr/local/share/icons/*
+      localizations:
+        - usr/local/share/locale/*
+      manpages:
+        - usr/local/share/man/*
+      pixmaps:
+        - usr/local/share/pixmaps/*
+      resources:
+        - usr/local/share/poedit/*
+    stage:
+      - $executables
+      - $freedesktop-desktop-entries
+      - $manpages
+      - $localizations
+      - $pixmaps
+      - $resources
+    prime:
+      - $executables
+      - $freedesktop-desktop-entries
+      - $manpages
+      - $localizations
+      - $pixmaps
+      - $resources
+    after:
+      - boostlib
+      - desktop-gtk3
+      - wxwidgets
+  boostlib:
+    plugin: dump
+    source: ../../deps/boost
+    source-type: local
+    build-packages:
+      # FIXME: Extracted from debian package, might be more than enough
+      - dctrl-tools
+      - zlib1g-dev
+      - libbz2-dev
+      - libicu-dev
+      - mpi-default-dev
+      - bison
+      - flex
+      - docbook-to-man
+      - help2man
+      - xsltproc
+      - g++
+      # - g++-6
+      - python
+      - python-all-dev
+      - python3
+      - python3-all-dev
+    override-build: |
+      ./bootstrap.sh
+      sudo ./b2 install\
+        -j $(nproc)\
+        --build-type=minimal\
+        --layout=system\
+        --with-iostreams\
+        --with-regex\
+        --with-system\
+        --with-thread\
+        threading=multi
+    filesets:
+      shared-libs:
+        - usr/local/lib/*.so*
+    stage:
+      - $shared-libs
+    prime:
+      - $shared-libs
+  wxwidgets:
+    plugin: dump
+    source: ../../deps/wx
+    source-type: local
+    build-packages:
+      # FIXME: Extracted from debian package, might be more than enough
+      - gettext
+      - libgtk2.0-dev
+      - g++
+      - zlib1g-dev
+      - libjpeg-dev
+      - libpng-dev
+      - libtiff5-dev
+      - libsm-dev
+      - libgl1-mesa-dev
+      - libglu1-mesa-dev
+      - autotools-dev
+      - autoconf
+      - libexpat1-dev
+      - libxt-dev
+      - libgstreamer1.0-dev
+      - libgstreamer-plugins-base1.0-dev
+      - libgtk-3-dev
+      - libwebkit2gtk-4.0-dev
+      - libnotify-dev
+    override-build: |
+      ./configure
+      make -j$(nproc)
+      sudo make install
```

Comment:
There's no reason build should use `sudo`, I'm pretty sure.

Reply:
This will be fixed in the next push.